org.eclipse.xtend.lib.tests/src/org/eclipse/xtend/lib/annotations/DataTest.xtend

```diff
@@ -2,19 +2,26 @@ package org.eclipse.xtend.lib.annotations
 
 import org.junit.Test
 import org.junit.Assert
+import org.eclipse.xtend2.lib.StringConcatenation
 
 class DataTest {
 	
 	@Test def void testData() {
-		Assert.assertEquals('''
+		Assert.assertEquals(toUnix('''
 			MyDataClass [
 			  arg = "foo"
 			  foo = false
-			]'''.toString, new MyDataClass("foo").toString)
+			]'''), new MyDataClass("foo").toString)
 	}
 	
 	@Data static class MyDataClass {
 		String arg
 		boolean foo = false
 	} 
+
+	def static String toUnix(String s) {
```

Comment:
Can we use org.eclipse.xtext.util.Strings.toUnixLineSeparator(CharSequence) instead to have a consistent solution.

Reply:
Using that method would introduce a new dependency from `org.eclipse.xtend.lib.tests` to `org.eclipse.xtext.util`, i.e. from xtext-lib to xtext-core. That would be bad, woundn't it?

Reply:
True. Looks goods to me.